server/trillian_log_server/main.go

```diff
@@ -60,6 +58,8 @@ var (
 	mySQLURI        = flag.String("mysql_uri", "test:zaphod@tcp(127.0.0.1:3306)/test", "Connection URI for MySQL database")
 	rpcEndpoint     = flag.String("rpc_endpoint", "localhost:8090", "Endpoint for RPC requests (host:port)")
 	httpEndpoint    = flag.String("http_endpoint", "localhost:8091", "Endpoint for HTTP metrics and REST requests on (host:port, empty means disabled)")
+	tlsCertFile     = flag.String("tls_cert_file", "", "Path to the TLS server certificate.")
```

Comment:
Consider adding a bit more text to the description here too, about running a non-TLS server if you don't specify these flags.

Reply:
Done.